src/SFA.DAS.ApplyService.InternalApi.UnitTests/RoatpGatewayOrganisationChecksControllerTests.cs

```diff
@@ -0,0 +1,104 @@
+
+            var returnedPage = new Page { PageOfAnswers = new List<PageOfAnswers> { new PageOfAnswers { Answers = new List<Answer> { 
+                                        new Answer { QuestionId = RoatpPreambleQuestionIdConstants.UkrlpLegalAddressLine1, Value = "First Address"},
+                                        new Answer { QuestionId = RoatpPreambleQuestionIdConstants.UkrlpLegalAddressLine2, Value = "Second Address" },
+                                        new Answer { QuestionId = RoatpPreambleQuestionIdConstants.UkrlpLegalAddressLine3, Value = "Third Address" },
+                                        new Answer { QuestionId = RoatpPreambleQuestionIdConstants.UkrlpLegalAddressLine4, Value = "Fourt Address" },
+                                        new Answer { QuestionId = RoatpPreambleQuestionIdConstants.UkrlpLegalAddressTown, Value = "Coventry" },
+                                        new Answer { QuestionId = RoatpPreambleQuestionIdConstants.UkrlpLegalAddressPostcode, Value = "CV1 2WT" }
+            } } } };
+
+            _qnaApiClient.Setup(x => x.GetPageBySectionNo(applicationId, preambleSequenceNo, preambleFirstSectionNo, RoatpWorkflowPageIds.Preamble)).ReturnsAsync(returnedPage);
+
+            var responseGetOrganisationAddress = _controller.GetOrganisationAddress(applicationId).GetAwaiter().GetResult().Result;
+            var response = responseGetOrganisationAddress as OkObjectResult;
+            var responseContactAddress = response.Value as ContactAddress;
+
+            Assert.AreEqual(expectedContactAddress.Address1, responseContactAddress.Address1);
+            Assert.AreEqual(expectedContactAddress.Address2, responseContactAddress.Address2);
+            Assert.AreEqual(expectedContactAddress.Address3, responseContactAddress.Address3);
+            Assert.AreEqual(expectedContactAddress.Address4, responseContactAddress.Address4);
+            Assert.AreEqual(expectedContactAddress.Town, responseContactAddress.Town);
+            Assert.AreEqual(expectedContactAddress.PostCode, responseContactAddress.PostCode);
+        }
+
+        [Test]
+        public void OrganisationChecks_ico_number_is_returned()
+        {
+            var applicationId = Guid.NewGuid();
+            var expectedIcoNumber = "TIG8ZZTQ";
+
+            var returnedPage = new Page
+            {
+                PageOfAnswers = new List<PageOfAnswers> 
+                                    { new PageOfAnswers 
+                                            { Answers = new List<Answer> 
+                                                            {
+                                                                new Answer 
+                                                                    { 
+                                                                        QuestionId = RoatpYourOrganisationQuestionIdConstants.IcoNumber, 
+                                                                        Value = "TIG8ZZTQ"
```

Comment:
Could you reuse the expectedIcoNumber variable here?

Reply:
It's changed now and expectedIcoNumber  is reused